src/__internal__/validations/validation-icon.component.js

```diff
@@ -87,6 +87,7 @@ const ValidationIcon = ({
         }
         isPartOfInput={isPartOfInput}
         inputSize={size}
+        blockFocusBehaviour
```

Comment:
This props is not clear, 

`blockFocusBehaviour=true` allows the icon to be focused.

I think we should run this by @tempertemper - is it necessary for the icon to be focusable, couldn't we use [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute) and setting [`aria-invalid=false`](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21.html) if the `error` prop is passed (warnings shouldn't fail validation).

**New Behaviour**

https://user-images.githubusercontent.com/2328042/135587809-e2b957a1-b335-4f13-be14-b11cfb4f9e40.mov

**Existing Behaviour**

https://user-images.githubusercontent.com/2328042/135587828-96a1e0d5-2515-44b6-affb-3b48065f1095.mov

Reply:
Ah I missed that, yeah happy to defer to the expert and implement whatever is deemed the right approach

Reply:
I'll add the `DO NOT MERGE` label for now